zen/xwayland-view.c

```diff
@@ -44,6 +44,20 @@ zn_xwayland_view_wlr_xwayland_surface_destroy_handler(
   zn_xwayland_view_destroy(self);
 }
 
+static void
+zn_xwayland_view_impl_focus(struct zn_view* view)
+{
+  struct zn_xwayland_view* self = zn_container_of(view, self, base);
+  wlr_xwayland_surface_activate(self->wlr_xwayland_surface, true);
```

Comment:
```suggestion
  wlr_xwayland_surface_activate(self->wlr_xwayland_surface, true);
  wlr_xwayland_surface_restack(surface, NULL, XCB_STACK_MODE_ABOVE);
```

We don't need `wlr_xwayland_surface_restack`?

Reply:
I'm going to implement about restack as another PR.